### PR TITLE
Visited stories

### PR DESCRIPTION
**Pull request by vycius**
screenshot_20161126-202610

* Save visited stories to Realm db;
* Show different markers for visited stories;
* Let user always click on visited stories markers;

I have an idea to implement user collected stories list https://github.com/gyvosistorijos/gyvos-istorijos-android/issues/23 after this

### CODE REVIEW

File: app/src/main/java/lt/gyvosistorijos/entity/VisitedStory.kt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package lt.gyvosistorijos.entity
+
+import java.util.*
+
+class VisitedStory(
+        val id: String,
+        val date: Date,
+        val story: Story
```
Comment (justasm) [inline app/src/main/java/lt/gyvosistorijos/entity/VisitedStory.kt]: `VisitedStory` doesn't need a story object.

**Review by justasm — CHANGES_REQUESTED** [review]
I get a crash on launch due to a missing migration.
```
io.realm.exceptions.RealmMigrationNeededException: RealmMigration must be provided
// ...
Caused by: io.realm.exceptions.RealmMigrationNeededException: The 'VisitedStoryRealm' class is missing from the schema for this Realm.
```

**Comment by vycius** [comment]
Thanks. I will fix migration issue and remove `Story` from `VisitedStory`. Could you give color codes for visited place marker ?

**Comment by justasm** [comment]
Since we're using cold colours for the markers you could try a softer cold colour e.g. B8EEDE and 87AEA3

**Review by justasm — APPROVED** [review]
Will have to look into proper migrations in the future 🙂

**Pull request merged**